rr/gstreamer/gst_media.py

```diff
@@ -158,7 +172,11 @@ def get_triggers(self):
 
     @classmethod
     def make(cls, desc, all_triggers):
-        pipe = 'uridecodebin uri=%s caps=video/x-h264 ! queue ! h264parse ! avdec_h264 ! queue ! videoconvert ! videoscale ! video/x-raw,width=320,height=240,format=RGB ! appsink emit-signals=true name=appsink' % (
+        pipe = '''uridecodebin uri=%s caps=video/x-h264 ! queue !
+                    h264parse ! v4l2h264dec capture-io-mode=5 ! video/x-raw,format=NV12 ! queue !
```

Comment:
Does this indentation come from autopep8?

if not make the beginning of both lines match